docs/usage/shoot_istio_dns_certs.md

```diff
@@ -0,0 +1,483 @@
+---
+title:  Gardener yourself a Shoot with Istio, custom Domains, and Certificates
```

Comment:
Why do you think this guide belongs to g/g?
Probably the [DNS](https://github.com/gardener/gardener-extension-shoot-dns-service) and [Cert](https://github.com/gardener/gardener-extension-shoot-cert-service) extensions are better candidates for it.